src/Components/Form.js

```diff
@@ -0,0 +1,143 @@
+
+import { useState } from "react";
+
+const Form = ({getActivity}) => {
+   
+   const [userNumberChoice, setUserNumberChoice] = useState("1");
+   const [userActivityChoice, setUserActivityChoice] = useState("")
+   const [newSearch, setNewSearch] = useState("Show me the activity");
+
+   const handleChange = (event) => {
+      setUserNumberChoice(event.target.value);
+      setNewSearch("Show me the activity")
+      console.log(event.target.value)
+   };
+
+   const handleTypeChange = (event) => {
+      setUserActivityChoice(event.target.value)
+   }
+
+   
+
+   return (
+      <div>
+         <form 
+         onSubmit={(event) => {
+            event.preventDefault();
+            setNewSearch("Give me more of that")
+            getActivity(userNumberChoice, userActivityChoice)
+            }
+         } 
+         className="user-form" 
+         name="user-selection">
+            <fieldset>
+               <legend>How many kids are feeling bored?</legend>
+               
+               <div>
+                  <input 
+                     type="radio" 
+                     name ="number-of-kids" 
+                     id="one-kid" 
+                     value="1" 
+                     checked={userNumberChoice === '1'}
+                     onChange={handleChange}></input>
+                  <label htmlFor="one-kid">1</label>
+               </div>
+               <div>
+                  <input 
+                     type="radio" 
+                     name ="number-of-kids" 
+                     id="two-kids" 
+                     value="2"
+                     checked={userNumberChoice === '2'}
+                     onChange={handleChange}></input>
+                  <label htmlFor="two-kids">2+</label>
+               </div>
+            </fieldset>
+            <fieldset>
+               <legend>What kind of activity do you have in mind?</legend>
```

Comment:
Admittedly, I got this feedback from Adrian, but I think it's good.

For future-proofing the app (maybe BoredAPI adds more categories, or renames them), you can create an array of the "type-of-activity" activities, use a .map function and append the values in the HTML instead of hard-coding them.

How? I need to look into that, too... lol.